specs/phase0/p2p-interface.md

```diff
@@ -856,6 +856,10 @@ Clients MUST support requesting blocks since the latest finalized epoch.
 Clients MUST respond with at least one block, if they have it.
 Clients MAY limit the number of blocks in the response.
 
+Clients MAY include a block in the response as soon as it passes the gossip validation rules.
+Clients MUST NOT respond with blocks that fail gossip validation rules.
+Clients SHOULD NOT include blocks that fail `state_transition(block)`.
```

Comment:
Nit: `state_transition` has more arguments than just a block.
(this is probably my fault as I have used it this way in one of my comments)
```suggestion
Clients SHOULD NOT respond with blocks that fail beacon chain state transition.
```